assets/js/utils.js

```diff
@@ -62,4 +62,13 @@ function setDateToday() {
     document.getElementById("datepicker").value = [today.getFullYear(), month, day].join("-");
 }
 
-export { locateMe, setDateToday, preciseLocation }
+function randomPriceRange() {
```

Comment:
Maybe just put this function in search.js file? Other pages may not have the elements.

Reply:
Addressed in f5bc0b